src/QuadForm/Lattices.jl

```diff
@@ -305,8 +305,8 @@ function generators(L::AbstractLat; minimal::Bool = false)
     d = ncols(St)
     for i in 1:nrows(St)
       if base_ring(L) isa NfOrd
-        I = numerator(St.coeffs[i])
-        den = denominator(St.coeffs[i])
+        I = numerator(coefficient_ideals(St)[i])
```

Comment:
Please check that this does not make a copy

Reply:
No copy, access directly `St.coeffs`